backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/iinvoicecandidate/I_Invoice_Candidate_StepDef.java

```diff
@@ -137,10 +137,10 @@ public void loadImportInvoiceCandidatesByPartnerValue(@NonNull final DataTable d
 		}
 	}
 
-	@And("metasfresh initially has no I_Invoice_Candidate data")
-	public void delete_I_Invoice_Candidate_data()
+	@And("^metasfresh initially has no (I_.*) import data$")
+	public void delete_I_Invoice_Candidate_data(@NonNull final String tableName)
```

Comment:
i think now this method no longer belongs here

Reply:
moved to `de.metas.cucumber.stepdefs.dataImport.C_DataImport_StepDef`.